rep-2000.rst

```diff
@@ -306,6 +307,109 @@ Build System Support:
+|  rmw_connext_cpp         | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_opensplice_cpp       | ADLink OpenSplice   | Tier 2        | All Platforms except Debian | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-RTPS  | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minumum language requirements:
+
+- C++14
+- Python 3.5
+
+
+Dependency Requirements:
+
++-------------+-------------------------------------------------+---------------------+
+|             |                        Required Support         | Recommended Support |
++-------------+----------------+---------------+----------------+---------------------+
+| Package     | Ubuntu  Bionic |     MacOS**   |   Windows 10** | Debian Stretch [s]  |
++=============+================+===============+================+=====================+
+| CMake       |     3.10.2     |     3.12.2    |     3.12.1     |        3.7.2        |
++-------------+----------------+---------------+----------------+---------------------+
+| EmPY        |     3.3.2      |     3.3.2     |     3.3.2      |        3.3.2        |
++-------------+----------------+---------------+----------------+---------------------+
+| Ogre        |                                      1.10*                            |
++-------------+----------------+---------------+----------------+---------------------+
+| OpenCV      |     3.2.0      |     3.4.1     |     3.4.6*     |        3.2*         |
++-------------+----------------+---------------+----------------+---------------------+
+| Poco        |     1.8.0      |     1.9.0     |     1.8.0*     |        1.8.0*       |
++-------------+----------------+---------------+----------------+---------------------+
+| Python      |     3.6.5      |     3.7.3     |     3.7.3      |        3.5.3        |
++-------------+----------------+---------------+----------------+---------------------+
+| Qt          |     5.9.5      |     5.11.2    |     5.10.0     |        5.7.1        |
++-------------+----------------+---------------+----------------+---------------------+
+|                              |           Linux only                                 |
```

Comment:
This "Linux only" line is a bit lost here, is it meant to refer to PCL below? (that's what it looks like on Bouncy, and it's equally strange on Crystal).

I's suggest to make `Linux only` and `RMW DDS Milddleware Providers` bold so it's clear that they're headers